.github/workflows/nightly-test.yaml

```
@@ -0,0 +1,61 @@
+name: Nightly Latest/Edge Tests
+
+on:
+  # TODO: remove, this is just testing
+  push: 
+    branches: 
+      - KU-91/nightly-runs
```

Can we run this on release branches too??

```suggestion
    branches: 
      - KU-91/nightly-runs
      - main
      - 'release-[0-9]+.[0-9]+'
```

---

We sure can, I don't think we want to run this every time we push to the main branch though? We'd be running the integration test twice?

---

OH RIGHT not on push here... i was hoping to run the cron on every branch that matches main or release-1xx